new/LIT7301QeneBalelit.xml

```diff
@@ -0,0 +1,77 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7301QeneBalelit" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">በሌሊት፡ ለሞዓብ፡ አመ፡ ያጠፍአ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Ba-lelit la-Moʿāb ʾama yāṭaffǝʾa (qǝne)</title>
```

Comment:
I would stay with sellase-type, if it words out and no emendations would be needed for this.